wordpress-mobile/WordPress-iOS
# [Gutenberg] Hotfix 1.15.2

PR to merge hotfix v1.15.2 into `release/13.5`

`gutenberg-mobile` PR: https://github.com/WordPress/gutenberg/pull/18128

**To test:**

- Upload media (Image, Video, Media&Text Blocks).
- Copy images from Photos app into gutenberg editor.
- Block inserter layout should be correct after device rotation. (https://github.com/wordpress-mobile/gutenberg-mobile/pull/1495)
- Added left right borders to inner blocks (https://github.com/wordpress-mobile/gutenberg-mobile/pull/1496)
- Removed alignment options from Media & Text until they are fixed (https://github.com/wordpress-mobile/gutenberg-mobile/pull/1500)

Update release notes:
